### ISSUE TITLE
Error when installing from extenions.gnome.org

### ISSUE THREAD

**Issue by WuerfelDev** [hermes83/compiz-alike-windows-effect#11]
I just received an update via extensions.gnome.org and installed it. Then it showed the red [ERROR] for this extension.
I tried reinstalling but still get the error.
Not sure why

But installing from GitHub worked fine

```
git clone https://github.com/hermes83/compiz-alike-windows-effect ./.local/share/gnome-shell/extensions/compiz-alike-windows-effect@hermes83.github.com
gnome-extensions enable compiz-alike-windows-effect@hermes83.github.com
busctl --user call org.gnome.Shell /org/gnome/Shell org.gnome.Shell Eval s 'Meta.restart(null)'
```

**Comment by Welcius** [comment]
Same behavior, updated the extension and got an error.

`Jul 10 10:24:36 ubu gnome-shell[2699]: JS ERROR: Extension compiz-alike-windows-effect@hermes83.github.com: Error: A different version was loaded previously. You need to log out for changes to take effect.#012loadExtension@resource:///org/gnome/shell/ui/extensionSystem.js:341:52#012callback@resource:///org/gnome/shell/ui/extensionDownloader.js:232:39#012gotExtensionZipFile/<@resource:///org/gnome/shell/ui/extensionDownloader.js:115:13
`
Did not try to logout though since I applied the solution that Wuerfel mentioned before.

**Comment by GustavoMHyLine** [comment]
Same in Ubuntu 20.04

**Comment by Ngugisenior** [comment]
Ubuntu 20.04 Broken update
![Screenshot from 2020-07-10 16-22-12](https://user-images.githubusercontent.com/25932594/87165293-3cf39480-c2ca-11ea-9ea2-ab78e2d1c57e.png)

**Comment by david-christiansen** [comment]
The update failed for me under Fedora 32, but restarting gnome-shell (with alt-f2 then `r`) fixed it for me.

**Comment by rtlewis88** [comment]
FWIW, the new update did install correctly (no errors or restarts) on openSUSE Leap 15.2 via GNOME 3.34.4. Also, FWIW, as an old Compiz addict, I get this version to precisely mimic the configuration I enjoyed on MATE, LXDE, and Xfce -  before totally switching to GNOME a couple of years ago. For some reason, I could not get the libanimation version to do do that. So, you can give this version some more credit for being wholly spot on! Thanks!

**Comment by dgmvecuador** [comment]
> The update failed for me under Fedora 32, but restarting gnome-shell (with alt-f2 then `r`) fixed it for me.

Restarting fixed the problem for me too in Ubuntu 20.04.

**Comment by WuerfelDev** [comment]
As said above it works after restarting the shell.
The author also noted that on the extension page too

(And IIRC i saw it with an other extension too, so it might be a shell or extension management problem)
